Makefile

```diff
@@ -4,7 +4,7 @@ bootstrap: | resize bootstrap_node_javascript bootstrap_environment bootstrap_do
 # The 2 slowest tasks are the cdk standing up the stacks
 # and the python install
 bootstrap_environment:
-	$(MAKE) bootstrap_cdk bootstrap_python bootstrap_java bootstrap_node_typescript -j 4
+	$(MAKE) bootstrap_cdk bootstrap_python bootstrap_java bootstrap_node_typescript bootstrap_csharp -j 5
```

Comment:
nit:  I think `_net` would be better for consistency (with our other docs and repo names) than `_csharp`, here and elsewhere.

Reply:
I renamed this and the directory to use `dotnet`.